inbox/src/main/java/org/egov/inbox/service/InboxService.java

```diff
@@ -84,10 +84,16 @@ public InboxResponse fetchInboxData(InboxSearchCriteria criteria, RequestInfo re
 		List<String> inputStatuses = new ArrayList<>();
 		if(!CollectionUtils.isEmpty(processCriteria.getStatus()))
 			inputStatuses = new ArrayList<>(processCriteria.getStatus());
-		// Since we want the whole status count map regardless of the status filter being passed
+		StringBuilder assigneeUuid = new StringBuilder();
+		if(!ObjectUtils.isEmpty(processCriteria.getAssignee())){
+			assigneeUuid = assigneeUuid.append(processCriteria.getAssignee());
+		}
+		// Since we want the whole status count map regardless of the status filter and assignee filter being passed
 		processCriteria.setStatus(null);
+		processCriteria.setAssignee(null);
```

Comment:
why is this needed?

Reply:
you are  setting processCriteria.setAssignee(null) and then again overwriting it with processCriteria.setAssignee(assigneeUuid.toString());